src/MakingSense.AspNetCore.Authentication.SimpleToken/SimpleTokenDefaults.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MakingSense.AspNetCore.Authentication.SimpleToken
+{
+	public static class SimpleTokenDefaults
+	{
+		public const string AuthenticationScheme = "SimpleToken";
```

Comment:
Should it not be Bearer? 

https://github.com/MakingSense/aspnet-authentication-simpletoken/pull/10/files#diff-1f3d419055594fba2c5c3d60ecec2ebcL36

(I do not know how it works)

Reply:
Checking this, I think you are right and should be Bearer